src/Frontend/Page/Topic.hs

```diff
@@ -97,43 +98,53 @@ instance ToHtml EditTopic where
 -- Maybe some buttons to hide?
 instance ToHtml ViewTopic where
     toHtmlRaw = toHtml
-    toHtml p@ViewTopicDelegations = semanticDiv p "ViewTopicDelegations" -- FIXME
+
+    toHtml p@(ViewTopicDelegations topic delegations) = semanticDiv p $ do
+        viewTopicHeaderDiv topic TabDelegation
+        -- related: Frontend.Page.User.renderDelegations
+        -- FIXME: implement!
+        pre_ $ topic ^. showed . html
+        pre_ $ delegations ^. showed . html
+
     toHtml p@(ViewTopicIdeas tab topic ideasAndNumVoters) = semanticDiv p $ do
-        -- assert tab /= TabDelegation
-        div_ [class_ "topic-header"] $ do
-            header_ [class_ "detail-header"] $ do
-                a_ [class_ "btn m-back detail-header-back", href_ $ U.Space space U.ListTopics] "Zu Allen Themen"
-                nav_ [class_ "pop-menu m-dots detail-header-menu"] $ do
-                    ul_ [class_ "pop-menu-list"] $ do
-                        li_ [class_ "pop-menu-list-item"] $ do
-                            a_ [id_ "edit-topic",  href_ . U.Space space $ U.EditTopic topicId] $ do
-                                i_ [class_ "icon-pencil"] nil
-                                "bearbeiten"
-            h1_   [class_ "main-heading"] $ do
-                span_ [class_ "sub-heading"] . toHtml $ phaseName phase
-                toHtml $ topic ^. topicTitle
-            p_ [class_ "sub-header"] $ topic ^. topicDesc . html
-            when (phase == PhaseRefinement) $
-                a_ [class_ "btn-cta heroic-cta", href_ . U.Space space $ U.CreateTopicIdea topicId] "+ Neue Idee"
-            when (phase < PhaseResult) .
-                a_  [class_ "btn-cta heroic-cta", href_ . U.Space space $ U.CreateTopicDelegation topicId] $ do
-                    i_ [class_ "icon-bullhorn"] nil
-                    "Stimme Beauftragen"
-            div_ [class_ "heroic-tabs"] $ do
-                tabLink topic tab TabAllIdeas
-                when (phase >= PhaseVoting) $ tabLink topic tab TabVotingIdeas
-                when (phase >= PhaseResult) $ tabLink topic tab TabWinningIdeas
-                tabLink topic tab TabDelegation
+        assert (tab /= TabDelegation) $ viewTopicHeaderDiv topic tab
```

Comment:
Generally I don't like asserts in code, but for the time being I can live with it.